docs/topics/proc_add-debug-logging-to-app-vertx.adoc

```diff
@@ -51,9 +51,9 @@ The default configuration establishes a single handler on the root logger for se
 
 
 
-A JUL logging configuration file can be specified in the normal JUL way by providing a system property called: java.util.logging.config.file with the value being your configuration file.
-For more information on this and the structure of a JUL config file please consult the JUL logging documentation.
+A `java.util.logging` logging configuration file can be specified in the normal `java.util.logging` way by providing a system property called: java.util.logging.config.file with the value being your configuration file.
+For more information on this and the structure of a `java.util.logging` config file please consult the `java.util.logging` logging documentation.
 
 Vert.x also provides a slightly more convenient way to specify a configuration file without having to set a system property.
-Just provide a JUL config file with the name vertx-default-jul-logging.properties on your classpath (e.g. inside your fatjar) and Vert.x will use that to configure JUL.
+Just provide a `java.util.logging` config file with the name vertx-default-`java.util.logging`-logging.properties on your classpath (e.g. inside your fatjar) and Vert.x will use that to configure `java.util.logging`.
```

Comment:
Woohoo, that's some search&replace here :-) The entire "vertx-default-`java.util.logging`-logging.properties" thing should be `code` I guess, or none of it.

Reply:
Ouch! that's pretty bad, sorry :D forgot to double check before pushing.
This shall be thoroughly cleaned up!